evaluate/repository_test.go

```diff
@@ -304,6 +413,10 @@ func TestRepositoryLoadConfiguration(t *testing.T) {
 		TestDataPath:   filepath.Join("..", "testdata"),
 		RepositoryPath: filepath.Join("golang", "plain"),
 
+		MutationBefore: func(t *testing.T, repositoryPath string) {
+			assert.NoError(t, os.Remove(filepath.Join(repositoryPath, "repository.json")))
+		},
+
```

Comment:
I think this needs to go into the previous commit where you added the config to the plain directory!